python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -104,3 +108,23 @@ def __init__(self, num_processes: int = 1,
 
             super().__init__(accelerator=accelerator,
                              plugins=[plugin], *args, **kwargs)
+
+    @staticmethod
```

Comment:
I think we can make it a member function

Reply:
Static method is more reasonable, I think, after all we are not using any Trainer instance in compile. And we can do either:
1.  `trainer.compile(model, loss, optimizer)` 
2.  `Trainer.compile(model, loss, optimizer)`